src/ui/components/map/providers/mapprovider.js

```diff
@@ -90,6 +90,25 @@ export default class MapProvider {
      * @type {boolean}
      */
     this._collapsePins = config.collapsePins || false;
+
+    /**
+     * Locale of the map. Defaults to the locale specified in ANSWERS.init() or to 'en'
+     * if a locale is not provided in the init.
+     * @type {string}
+     */
+    const locale = config.locale || config.initLocale || 'en';
+
+    /**
+     * Language of the map. Defaults to 'en'
+     * @type {string}
+     */
+    this._language = locale.substring(0, 2) || 'en';
+
+    /**
+     * Fallback which is used if _language is not supported
+     * @type {string}
+     */
+    this._languageFallback = 'en';
```

Comment:
My initial thought is we don't make this a member variable of the `MapProvider` super class as it only impacts the Mapbox implementation. Do you think we will need this for Google at some point?

Reply:
Good point. It's unlikely we'll need this for Google since google supports everything MapBox does and more. Updating